setup.cfg

```diff
@@ -1,7 +1,9 @@
 [metadata]
 name = anyGPT
 version = 0.0.1a1
-description = A general purpose library for training any type of GPT model. Support for gpt-1, gpt-2, and gpt-3 models.
+description = A general purpose library for training any type of GPT model.
+long_description = Support for gpt-1, gpt-2, and gpt-3 models.
```

Comment:
Use the same description for both description and long description for now until we find an automated way of populating long description from markdown.